src/Docusaurus/docs/about/system.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 1
+---
+
+# System requirements
+
+## Running packages built with WiX
+
+In general, packages you build with WiX will work on Windows 7 or later. Code that you introduce -- for example, custom actions or bootstrapper applications -- can require later versions of Windows.
+
+
+## Building packages with WiX
+
+WiX v4 is built on .NET 6 and so inherits its system requirements. 
```

Comment:
Technically it's .NET 6 or NETFX 4.7.2.

Reply:
👍 But with your latest, I'm getting...

```
[ERROR] Docusaurus server-side rendering could not render static page with path /404.html.
[INFO] It looks like you are using code that should run on the client-side only.
To get around it, try using `<BrowserOnly>` (https://docusaurus.io/docs/docusaurus-core/#browseronly) or `ExecutionEnvironment` (https://docusaurus.io/docs/docusaurus-core/#executionenvironment).
It might also require to wrap your client code in `useEffect` hook and/or import a third-party library dynamically (if any).
```

Reply:
That kinda makes sense. I'll poke at it later. Never saw that myself, interview

Reply:
Go with this and I'll resolve the `<BrowserOnly>` issue.